### PR TITLE
Add file associations and icons

### LINKED ISSUE #1031: As a xoder I want XOD file associations so that IDE feels native and quicker to work with
### Rationale

It’s so frustrating that I can’t open QCad files by simply double-clicking on them in my file manager :sob:

### Icons

[xod-file-icons.zip](https://github.com/xodio/xod/files/1643364/xod-file-icons.zip)

### Acceptance criteria

- [ ] The file associations are set up properly upon install on Win, macOS, Linux; icons are applied
- [ ] Double-clicking a `project.xod` opens that project
- [ ] Double-clicking a `my-patch.xodp` opens a project the patch belongs to
- [ ] Double-clicking a `hello.xodball` imports the xodball (into the current workspace)

### PR DESCRIPTION
**Pull request by brusherru**
It closes #1031.

### CODE REVIEW

File: packages/xod-client-electron/package.json [hunk 1]
```diff
@@ -91,6 +91,24 @@
...
       ]
     },
+    "nsis": {
+      "oneClick": false,
+      "perMachine": false
+    },
+    "fileAssociations": [
+      {
+        "ext": "xod",
+        "description": "File of a XOD Project with meta information"
```
Comment (nkrkv) [inline packages/xod-client-electron/package.json]: XOD Project Manifest

File: packages/xod-client-electron/package.json [hunk 2]
```diff
@@ -91,6 +91,24 @@
...
+      },
+      {
+        "ext": "xodp",
+        "description": "File of a XOD Project with Patch contents"
```
Comment (nkrkv) [inline packages/xod-client-electron/package.json]: XOD Patch

File: packages/xod-client-electron/package.json [hunk 3]
```diff
@@ -91,6 +91,24 @@
...
+      },
+      {
+        "ext": "xodball",
+        "description": "Packed all-in-one XOD Project"
```
Comment (nkrkv) [inline packages/xod-client-electron/package.json]: XOD Packed Project

**Comment by brusherru** [comment]
@nkrkv done!

**Pull request merged**